README.md

```diff
@@ -27,17 +27,34 @@ This will ask you a few questions and then generate a project skeleton for you.
 
 # Requirements
 
-You'll need the following on all OSes:
+### Operating Systems
 
-* [Node](http://nodejs.org) v4 or later;
-* [Rust](http://rust-lang.org) v1.7 or later;
-* [multirust](https://github.com/brson/multirust) (only required for Neon projects that override the system default Rust).
+| Linux  | macOS | Windows |
+| ------ | ----- | ------- |
+| x      | x     | soon    |
 
-For Mac OS X, you'll need:
+For macOS, you'll need:
 
 * OS X 10.7 or later;
 * [XCode](https://developer.apple.com/xcode/download/).
 
+Windows support is on the way. Follow [#122](https://github.com/neon-bindings/neon/pull/122#issuecomment-268957333) to track the progress.
+
+### Rust and Node
+
+|              | Node 4 | Node 6 | Node 7 |
+| ------------ | ------ | ------ | ------ |
+| Rust stable  | x      | x      | x      |
+| Rust beta    | x      | x      | x      |
+| Rust nightly |        |        |        |
+
+Support for Rust stable and beta are expected. We do run builds against nightly, but allow them to fail.
```

Comment:
This doesn't state a minimum Rust stable version. We talked in #149 about our Node version policy but not our Rust version policy.

Rust seems stricter about backwards compatibility than Node, so maybe we'll be able to get away with just stating a minimum Rust stable version. I'm sure we can't realistically test on all Rust versions > _min_, but we should at least **state** a minimum. We could say something like:

> Although Neon is only tested on the latest stable Rust, it will likely work on a range of stable versions of Rust. It is known to require at least Rust 1.13.

(I think we should bump our lower bound from 1.7 to 1.13 so that our programming style can rely on `?` now.)

Let's go with the above verbiage for now, unless you think there's more we could do (maybe a small smoke test for the minimum Rust version in our CI so we can at least do a sanity check whether our recommended minimum is still plausible?) and merits another RFC bug.